RELEASE.md

```diff
@@ -199,21 +199,26 @@ To make a macOS release, go to macOS build machine:
 
 - Build machine must have:
   - Apple-trusted `Developer ID Application: Freedom of the Press Foundation (94ZZGGGJ3W)` code-signing certificates installed
+- Apple account must have:
+  - A valid application password for `altool` in the Keychain. You can verify
+    this by running: `xcrun altool --notarization-history 0 -u "<email>" -p "@keychain:altool"`
+  - Agreed to any new terms and conditions. You can find those if you visit
+    https://developer.apple.com and login with the proper Apple ID.
```

Comment:
Should we move this up a bit for the "Pre-release section". It feels like stuff we can verify before we start any actual release procedures.

Reply:
I agree, with an asterisk. Since the checklist is a bit verbose, I propose adding a brief item in the QA section, that points to the macOS release section. Check out 6d4e5cab0172f75f284a9db472b323464c694844.